tests/openwisp2/sample_monitoring/migrations/0006_create_chart_alertsettings.py

```diff
@@ -0,0 +1,130 @@
+# Generated by Django 3.0.3 on 2020-07-10 08:40
+
```

Comment:
we do not need to add migrations to the sample app, we can just destroy everything and recreate them.
@nepython @atb00ker @PabloCastellano @NoumbissiValere @pandafy :point_up_2: :warning: :warning: 
Please read this. I have seen you all doing it. Focus on the main migrations, sample app is not supposed to be maintained, it's only an example and used mainly for testing.